### ISSUE TITLE
Dual Wield - Swapping Main Slot 1h Weapon With Another Unequips Sub Slot 1h Weapon

### ISSUE THREAD

**Issue by FariskaEden** [EdenServer/community#5647]
### Checklist

- [X] I have searched for existing issues for issues like this one. The issue has not been posted.
- [X] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [X] I have provided my client version in the details. (type /ver into your game window)
- [X] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Currently when swapping a 1h main slot weapon while dual wielding for another 1h main slot weapon, the offhand 1h weapon will be unequipped. You should be able to swap a 1h main slot weapon with another 1h weapon without having the sub slot 1h weapon unequip. However if the main slot 1h weapon is swapped with a 2h weapon or fully unequipped, the sub slot 1h weapon should be unequipped then.

https://i.gyazo.com/fb4abb7699f579b8451d81c9c2a2fa33.mp4 - shows main slot swap without unequipping sub slot on retail

version 30181205_0

**Comment by Steam-arch** [comment]
I feel like Shield used to unequip if you swapped sword on era, but its been so long im not sure at all. They might have ironed in some small QOL things like that..? For example I swear meditate used to cancel/stop when I clicked on nyzul chest also but on current Retail it doesnt cancel on clicking chest/going up. Think ashcroftings also remembers it cancelling as I asked in ls about it awhile ago.

Tldr: are we sure it didnt unequip during era?

**Comment by dowzer** [comment]
This always annoyed me and I swear it didn't unequip the sub hand in era. Do we have any proof that it was supposed to unequip the sub hand?

**Comment by Steam-arch** [comment]
![image](https://github.com/EdenServer/community/assets/69089249/c7b22f21-6a77-4b28-a584-18ec28dbd8ef)

**Comment by falzum** [comment]
Dual wielding definitely always let you swap one or the other without unequipping your sub

**Comment by Steam-arch** [comment]
Maybe a ghost memory.

![image](https://github.com/EdenServer/community/assets/69089249/ca1d5049-3683-41ca-974a-17ad4337da43)


Swapping swords in november 08 didnt unequip shield atleast.
https://www.youtube.com/watch?v=V4hdowE94uw

**Comment by 59blargedy** [comment]
Resolved by another dev in August 7, 2023 update
